docs/operator-manual/notifications/services/teams.md

```diff
@@ -113,7 +113,7 @@ template.app-sync-succeeded: |
 
 ### summary field
 
-You can set a summary of the message that will be shown on Notifcation & Activity Feed 
+You can set a summary of the message that will be shown on Notification & Activity Feed 
```

Comment:
These docs are pulled from https://github.com/argoproj/notifications-engine. Would you please open a PR there?
```suggestion
You can set a summary of the message that will be shown on Notifcation & Activity Feed 
```